cmd/snap/cmd_snap_op.go

```diff
@@ -202,18 +203,24 @@ func refreshAll() error {
 	if err != nil {
 		return fmt.Errorf("cannot list updates: %s", err)
 	}
+	// nothing to update/list
+	if len(updates) == 0 {
+		return nil
```

Comment:
We agreed to have list showing the headers as usual for the zero case. I think we should do the same here?